### ISSUE TITLE
Bug - quest: Luthor's key

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#185]
```
Before posting, please read the following guidelines for posts in the issue
tracker:
http://code.google.com/p/andors-trail/wiki/Forums_vs_issuetracker


What steps will reproduce the problem?
1. when talking with Athamyr, he wants some cooked meat.
   ok, i answer "yes i have", and works, he gave me exp and permit
   to access the catacomb, but i DONT have any cooked meat in my
   inventory, i have not buyed in tavern...

What is the expected output? What do you see instead?
I expect to have to buy the meat and come back to Athamyr to give him

What version of the product are you using? On what device?
0.6.8 - htc desire 2.2

Please provide any additional information below.


```

Original issue reported on code.google.com by `liogiu@gmail.com` on 17 Apr 2011 at 6:36

**Comment by GoogleCodeExporter** [comment]
```
This is another bug in the previous Italian translation. I will write a script 
that compares conversation flow and required quest items, to check for these 
types of situations.
```

Original comment by `oskar.wi...@gmail.com` on 4 May 2011 at 5:26
- Changed state: **Started**
- Added labels: **Milestone-0.6.9**

**Comment by GoogleCodeExporter** [comment]
```
I made a quick script that corrected most of the previous Italian conversation 
translations, to make them have the same conversation flow as the English 
translation.

Committed in r75 .

However, there were 35 lines of dialogue that my script could not translate 
automatically. I have noted those place with the text UNTRANSLATED in the 
conversationlist.xml file.

liogiu, could you go through this file and translate the 35 lines where it says 
"UNTRANSLATED"?
http://code.google.com/p/andors-trail/source/browse/trunk/AndorsTrail/res/values
-it/conversationlist.xml

Thanks in advance,
/Oskar
```

Original comment by `oskar.wi...@gmail.com` on 5 May 2011 at 5:20

**Comment by GoogleCodeExporter** [comment]
```
Solved by issue 192.
```

Original comment by `oskar.wi...@gmail.com` on 7 May 2011 at 7:27
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:52
- Changed state: **Done**
